docs/UserGuide.md

```diff
@@ -110,20 +117,22 @@ The following is an annotated breakdown of **MODTrek's** GUI:
 ![UI Breakdown](images/Ui-breakdown.png)
 <p align="center"><em>Breakdown of GUI</em><br></p>
 
-1. **Command Input:** User inputs commands here.
+1. **Command Input:** This is where you input your commands. Press enter to execute them.
 2. **Command Screen:** Relevant messages from the application will be displayed here. This includes success messages, error
-messages and resul
-3. **Results Section:** This panel shows the degree progression of the user by default. This panel can be changed using the `view` command to toggle between the module list in **MODTrek** or the user's degree progression. Using the `find` command when making a query on the module list will navigate you to the module search screen.
+messages and results.
+3. **Results Section:** This panel shows your degree progression by default. This panel can be changed using the `view` command
+to toggle between the module list in **MODTrek** or the user's degree progression. Using the `find` command when making 
+a query on the module list will navigate you to the module search screen.
 
 The following is the annotated breakdown of the Results Section of **MODTrek's** GUI:
 ![UI subsection](images/Ui-subsections-detailed.png)
 <p align="center"><em>Breakdown of Results Section</em><br></p>
 
-1. **Doughnut Chart:** This will be shown on the degree progress section. It shows the completion progress of the degree
-and individual sub requirements of the degree. Your total MCS completed and CAP are also displayed.
-2. **Module List:** This will be shown in both the module list section and module search section. It shows all the module in the list.
-3. **Module Group:** This will be shown in both the module list section and module search section. It shows the modules in
-each sub category when sorted.
+1. **Doughnut Chart:** This will be shown in the degree progress section. It shows your completion progress of the degree
```

Comment:
I have discussed with @Cheamybunny and we have decided that there's no need to introduce the ModuleList, ModuleGroup and ModuleCard components in the UG as they are intuitive enough to the user using the GUI. It is originally only meant for the DG to connect each UI component to its corresponding Java class.

In UG, we will just introduce the three screens of the Results Section: Degree Progress Screen, Module List Screen, Module Search Screen. @Cheamybunny will reflect this change in a future commit.

Reply:
I will do this tyvm